examples/demo_qemu/main/Kconfig.projbuild

```diff
@@ -0,0 +1,107 @@
+menu "Example Configuration"
+
+    config ESP_MAXIMUM_RETRY
+        int "Maximum retry"
+        default 5
+        help
+            Set the Maximum retry to avoid station reconnecting to the AP
+            unlimited when the AP is really inexistent.
+
+    config WG_PRIVATE_KEY
+        string "Wireguard Private Key"
+        default "gBzqhni+8zFtpmeUEbneHu6zs/mesRLmxVFzqDyFJlg="
```

Comment:
Could you add something obviously not valid here, e.g. like:
`AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA=`

Reply:
Oh, you would have to set it up at runtime for all peers.

In that case maybe then just add a comment, that his key is only for testing purposes and should never be used on any real environments.

Reply:
Added

Reply:
Also, I added comment in [README.md](https://github.com/XmasHackathon2023/esp_wireguard/pull/3/files#diff-80721d91fcd85d737d656978fab4b922b873d210eea59c0036f65e05b3543d1cR7)